pyads/utils.py

```diff
@@ -46,3 +46,8 @@ def wrapper(*args: Any, **kwargs: Any) -> Callable:
         return wrapper
 
     return decorator
+
+
+def decode_ads(message: bytes) -> str:
+    """Decode a string that in encoded in the format used by ADS."""
+    return message.decode("windows-1252").strip(" \t\n\r\0")
```

Comment:
Is this the general encoding Beckhoff uses for strings?

Reply:
I believe so, according to [this](https://infosys.beckhoff.com/english.php?content=../content/1033/tc3_plc_intro/9007201784068235.html&id=):

> A STRING constant is a string enclosed by single quotation marks. The characters are encoded according to the Windows 1252 character set. As a subset of Windows-1252, the character set of ISO/IEC 8859-1 is supported.